src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCommon.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.dts.fact.admin;
+
+final class AdminCommon {
```

Comment:
I think you haven't pull from the master for a while. This file is old and have been renamed. Can you merge the latest code from master first please

Reply:
Merged with latest code from master and the file removed.